backend/driver.pm

```diff
@@ -156,15 +154,12 @@ sub _send_json {
     my $token = myjsonrpc::send_json($self->{backend_process}->channel_in, $cmd);
     my $rsp   = myjsonrpc::read_json($self->{backend_process}->channel_out, $token);
 
-    unless (defined $rsp) {
-        # this might have been closed by signal handler
-        no autodie 'close';
-        close($self->{backend_process}->channel_out);
-        $self->{backend_process}->channel_out(undef);
-        $self->{backend_process}->stop;
-        return;
-    }
-    return $rsp->{rsp};
+    return $rsp->{rsp} if defined $rsp;
+    # this might have been closed by signal handler
+    no autodie 'close';
+    close($self->{backend_process}->channel_out);
+    $self->{backend_process}->channel_out(undef);
+    $self->{backend_process}->stop;
```

Comment:
I think this needs an explicit `return` now, as currently the return value of the `stop` method would be returned.